src/org/wikibase/Wikibase.java

```diff
@@ -0,0 +1,186 @@
+/**
+*  This program is free software; you can redistribute it and/or
+*  modify it under the terms of the GNU General Public License
+*  as published by the Free Software Foundation; either version 3
+*  of the License, or (at your option) any later version. Additionally
+*  this file is subject to the "Classpath" exception.
+*
+*  This program is distributed in the hope that it will be useful,
+*  but WITHOUT ANY WARRANTY; without even the implied warranty of
+*  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+*  GNU General Public License for more details.
+*
+*  You should have received a copy of the GNU General Public License
+*  along with this program; if not, write to the Free Software Foundation,
+*  Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+*/
+package org.wikibase;
+
+import java.io.IOException;
+
+import java.net.URLEncoder;
+import java.util.StringTokenizer;
+import java.util.regex.Pattern;
+
+import org.wikibase.data.Entity;
+import org.wikipedia.Wiki;
+
+public class Wikibase extends Wiki {
+    public Wikibase() {
+        super("www.wikidata.org");
+    }
+
+    /**
+     * Returns an entity identified by the title of a wiki page.
+     * 
+     * @param site
+     * @param pageName
+     * @return
+     * @throws IOException
+     * @throws WikibaseException
+     */
+    public Entity getWikibaseItemBySiteAndTitle(final String site, final String pageName)
+        throws IOException, WikibaseException {
+        final StringBuilder url = new StringBuilder(query);
+        url.append("action=wbgetentities");
+        url.append("&sites=" + site);
+        url.append("&titles=" + URLEncoder.encode(pageName, "UTF-8"));
+        url.append("&format=xml");

```

Comment:
[This is redundant](https://github.com/MER-C/wiki-java/blob/ae06a628985c9c78e1bd2b0d74d5de615cae8fcd/src/org/wikipedia/Wiki.java#L541). This problem occurs multiple times.